question28/css/news.css

```
@@ -0,0 +1,164 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+.container {
+    max-width: 1000px;
+    width: 100%;
+    margin: 30px auto;
+}
+.logout_btn {
+    text-align: right;
+}
+.logout_btn button {
+    padding: 10px 30px;
+    margin-bottom: 30px;
+    background: #eeeeee;
+    cursor: pointer;
+}
+.newsWrapper {
+    /* margin: 50px auto; */
+}
+.news_tabs {
+    display: -webkit-flex;
+    display: flex;
+    justify-content: center;
+}
+.news_tabs li{
+    flex-grow: 1;
+    padding: 10px;
+    list-style: none;
+    border: solid 1px #CCC;
+    text-align: center;
+    cursor: pointer;
+    font-size: 18px;
+    font-weight: bold;
+    width: 25%;
+}
+.contentsWrapper {
+    border:solid 1px #CCC;
+    border-top:none;
+    background:#eee;
+    padding: 20px 60px;
+}
+.tabContents {
+    display: -webkit-flex;
+    display: flex;
+}
+.contentsContainer {
+    display: none;
+}
+.news_tabs li.is-active-tab{
+    background:#00bcd4;
+    color:#FFF;
+    transition: all 0.2s ease-out;
+}
+.contentsContainer.is-active-content{
+    display:block;
+}
+.tabContents ul {
+    width: 60%;
+    padding-right: 50px;
+}
+.tabContents .imgWrapper{
+    width: 40%;
+    max-width: 400px;
+}
+.tabContents .imgWrapper img{
+    width: 100%;
+}
+.tabContents ul li{
+    color: #3f51b5;
+    margin-bottom: 10px;
+}
+.tabContents ul li a{
+    text-decoration: none;
+    color: #3f51b5;
+    position: relative;
+    line-height: 2;
+}
+.tabContents ul li a:hover{
+    border-bottom: 1px solid #3f51b5;
+}
+.comment_icon_num {
```

because other is camelCase
```suggestion
.commentIconNum {
```

---

Thank you for your review!
There was no regularity...As for the class name, since the snake case is the main one, so I adapted it.

commit: https://github.com/csakyo/homework/pull/42/commits/e7f11ae3ef93b93aa981120ed365ef0a875fd23a